Model/MediaAccountModel.php

```diff
@@ -216,6 +216,12 @@ public function getStats(
         $dateFormat = null,
         $canViewOthers = true
     ) {
+        if ($dateFrom == $dateTo) {
+            $dateTo = $dateTo->add(
+                \DateInterval::createFromDateString('1 day')
+            )->sub(\DateInterval::createFromDateString('1 second'));
```

Comment:
Nothing wrong with your code but I wanted to make you aware that you can combine them like this:
`$dateTo = $datTo->add( DateInterval::createFromDateString('1 day -1 second') );`

(P.S., my personal style is $dayValue >= $start && $dayValue < $end so you don't need to decrement the end at all, but that has been an on going debate. No one seems to care about the 6 minutes of data per year using BETWEEN causes.